### PR TITLE
[DSD-5479] update eSignet mosipid properties to use IDA plugin instead of mock-IDA plugin

### PR DESCRIPTION
**Pull request by vharsh**
**NOTE**: More changes might be required for this PR and this would need a good review before this gets merged in.

### CODE REVIEW

File: esignet-mosipid.properties [hunk 1]
```diff
@@ -44,11 +44,15 @@ mosip.esignet.authentication-expire-in-secs=120
...
   '${server.servlet.path}/authorization/auth-code'}
 
+#This property is used for captcha validation and allowed values are send-otp and pwd.
+# captcha validation is enabled for the auth-factors - otp, pwd, bio and pin.
+mosip.esignet.captcha.required=send-otp,pwd,bio,pin
 
-#mosip.esignet.captcha.required.auth-factors={'PWD'}
-mosip.esignet.captcha.required.auth-factors={}
+
+mosip.esignet.captcha.required.auth-factors={'PWD'}
```
Comment (balaji-alluru) [inline esignet-mosipid.properties:55]: we don't need this property anymore.

**Review by balaji-alluru — APPROVED** [review]

File: esignet-mosipid.properties [hunk 2]
```diff
@@ -280,7 +283,7 @@ mosip.esignet.discovery.key-values={'issuer': '${mosip.esignet.domain.url}' ,\
   \ 'display_values_supported' : ${mosip.esignet.supported.ui.displays}, \
   \ 'subject_types_supported' : { 'pairwise' }, \
   \ 'claims_supported' : {'name','address','gender','birthdate','picture','email','phone_number','individual_id'}, \
-  \ 'acr_values_supported' : {'mosip:idp:acr:static-code', 'mosip:idp:acr:generated-code', 'mosip:idp:acr:linked-wallet', 'mosip:idp:acr:biometrics'},\
+  \ 'acr_values_supported' : {'mosip:idp:acr:static-code', 'mosip:idp:acr:generated-code', 'mosip:idp:acr:linked-wallet', 'mosip:idp:acr:biometrics', 'mosip:idp:acr:knowledge'},\
```
Comment (challabeehyv) [inline esignet-mosipid.properties:286]: do we need knowledge in mosipid plugin

File: esignet-mosipid.properties [hunk 3]
```diff
@@ -191,21 +194,21 @@ mosip.esignet.supported-pkce-methods={'S256'}
...
-#spring.redis.port=6379
+spring.cache.type=redis
+spring.cache.cache-names=${mosip.esignet.cache.names}
+spring.redis.host=redis-master-0.redis-headless.redis.svc.cluster.local
+spring.redis.port=6379
 management.health.redis.enabled=false
 
 # 'simple' cache type is only applicable only for Non-Production setup
-spring.cache.type=simple
+spring.cache.type=redis
```
Comment (challabeehyv) [inline esignet-mosipid.properties:211]: can we keep it simple and comment this property

**Review by kaifk468 — APPROVED** [review]

File: esignet-mosipid.properties (hunk 3 above)
Comment (vharsh) [reply esignet-mosipid.properties:211]: I believe IDA-plugin should need a transaction ID, hence the redis cache.

**Comment by vharsh** [comment]
closing this PR in favour of https://github.com/mosip/mosip-config/pull/6686

**Pull request closed without merging**